tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.7.0, tc, 13096
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea
+
+1. Open Chrome browser.
```

Comment:
@Comandeer can you explain how do you see that? I can restrict entire test to one browser but I'm not sure that this is the case here.
Noting else came to my mind about improving this test to be more generic. I'd appreciate a little help or some idea which might help me enchance this part. :)